pv_site_api/main.py

```diff
@@ -319,53 +320,68 @@ def get_pv_estimate_clearsky(site_uuid: str, session: Session = Depends(get_sess
     """
     ### Gets a estimate of AC production under a clear sky
     """
+    clearsky_estimates = get_pv_estimate_clearsky_many_sites(site_uuid, session)
+    return clearsky_estimates[0]
+
+
+@app.get("/sites/clearsky_estimate", response_model=list[ClearskyEstimate])
+def get_pv_estimate_clearsky_many_sites(
+    site_uuids: str,
+    session: Session = Depends(get_session),
+):
+    """
+    ### Gets a estimate of AC production under a clear sky for multiple sites.
+    """
+
     if is_fake():
-        fake_sites = make_fake_site()
-        site = fake_sites.site_list[0]
+        sites = make_fake_site().site_list
     else:
-        site_exists = does_site_exist(session, site_uuid)
-        if not site_exists:
-            raise HTTPException(status_code=404)
-        site = site_to_pydantic(get_site_by_uuid(session, site_uuid))
-
-    loc = location.Location(site.latitude, site.longitude)
-
-    # Create DatetimeIndex over four days, with a frequency of 15 minutes.
-    # Starts from midnight yesterday.
-    times = pd.date_range(start=get_yesterday_midnight(), periods=384, freq="15min", tz="UTC")
-    clearsky = loc.get_clearsky(times)
-    solar_position = loc.get_solarposition(times=times)
-
-    # Using default tilt of 0 and orientation of 180 from defaults of PVSystem
-    tilt = site.tilt if site.tilt is not None else 0
-    orientation = site.orientation if site.orientation is not None else 180
-
-    irr = irradiance.get_total_irradiance(
-        surface_tilt=tilt,
-        surface_azimuth=orientation,
-        dni=clearsky["dni"],
-        ghi=clearsky["ghi"],
-        dhi=clearsky["dhi"],
-        solar_zenith=solar_position["apparent_zenith"],
-        solar_azimuth=solar_position["azimuth"],
-    )
+        site_uuids_list = site_uuids.split(",")
+        sites = get_sites_by_uuids(session, site_uuids_list)
+
+    res = []
+
+    for site in sites:
+        loc = location.Location(site.latitude, site.longitude)
+
+        # Create DatetimeIndex over four days, with a frequency of 15 minutes.
+        # Starts from midnight yesterday.
+        times = pd.date_range(start=get_yesterday_midnight(), periods=384, freq="15min", tz="UTC")
+        clearsky = loc.get_clearsky(times)
+        solar_position = loc.get_solarposition(times=times)
+
+        # Using default tilt of 0 and orientation of 180 from defaults of PVSystem
+        tilt = site.tilt if site.tilt is not None else 0
```

Comment:
```suggestion
        tilt = site.tilt or 0
```

It's also good the way you wrote it (more semantic!), just pointing out that this is possible in python! The sme would work for `orientation`.

Reply:
Oh thanks, I didn't know this was possible! Changed it to your suggestion—think it's more clear